contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -212,10 +214,18 @@ public VoldemortBuildAndPushJob(String name, azkaban.utils.Props azkabanProps) {
                 if (clusterURLs.contains(url)) {
                     throw new VoldemortException("the URL: " + url + " is repeated in the "+ PUSH_CLUSTER + " property ");
                 }
-                this.clusterURLs.add(url);
-                AdminClient adminClient = createAdminClient(url, fetchAllStoresXml);
-                this.adminClientPerCluster.put(url, adminClient);
-                this.closeables.add(adminClient);
+                try {
+                    AdminClient adminClient = createAdminClient(url, fetchAllStoresXml);
+                    this.clusterURLs.add(url);
+                    this.adminClientPerCluster.put(url, adminClient);
+                    this.closeables.add(adminClient);
+                } catch (Exception e) {
```

Comment:
If we simply skip the unreachable colo here, the user may get confusion since the unreachable colo may not have the latest data if some colo is temporarily not available from push job, but the push job does succeed.
The Azkaban logs might be too long for user to understand the status of each colo every time.